new downloads pages and assets

spivotron opened the pull request:
Hello,
This commit contains the styling updates for the following downloads pages:

- Driverless AI v. 1.4.1
- Driverless AI v. 1.3.1
- H2O Latest Stable Release
- H2O Nightly Bleeding Edge
- Sparkling Water

Each HTML file references the assets folder and CSS folder to retrieve images and css files.

michalkurka commented:
Hi @spivotron, thanks for the contribution. Can you please give us more information about what it is driving the addition?

swenconrad commented:
Tom and Michal,

I asked Henry to integrate the changes.

Swen

spivotron commented:
Hi Tom,

I've removed the irrelevant files for this repo and created the new directory to hold the static assets.  I updated make-dist.sh with what I believe is the correct command to copy static into target.  

Can you please review my latest commit?

tomkraljevic commented on h2o-dist/index.html:
```diff
@@ -1,15 +1,15 @@
-  <head>
-    <meta charset="utf-8">
+<!-- saved from url=(0060)http://h2o-release.s3.amazonaws.com/h2o/rel-xia/2/index.html -->
+<html class="no-js"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
```
This does not belong here.  There should not be any references to rel-xia in this file.

tomkraljevic commented on h2o-dist/index.html:
```diff
@@ -1,15 +1,15 @@
+    <meta name="viewport" content="width=device-width, height=device-height, initial-scale=1.0, user-scalable=0, minimum-scale=1.0, maximum-scale=1.0">
 
-    <link href='http://fonts.googleapis.com/css?family=Open+Sans:400,300' rel='stylesheet' type='text/css'>
+    <link href="./Latest Stable Release_files/css" rel="stylesheet" type="text/css">
```
There should not be any reference to "Latest Stable Release" here.
It should be ./static

tomkraljevic commented on h2o-dist/index.html:
```diff
@@ -367,42 +370,56 @@
     </script>
+    <link rel="stylesheet" type="text/css" href="static/css/main.css">
+    <link rel="stylesheet" type="text/css" href="static/css/bleeding-edge.css">
+    <link rel="stylesheet" href="https://use.typekit.net/gbq5zcz.css">
```
Should this be in ./static?

spivotron replied:
Typically, typekit stylesheets are linked to directly to typekit

spivotron replied:
Ok, I will put this in the static folder

tomkraljevic commented on h2o-dist/index.html:
```diff
@@ -367,42 +370,56 @@
-        <p class="version">Version SUBST_PROJECT_VERSION</p>
+      <div id="header">
+          <a href="http://h2o.ai/" id="logo">
+            <img src="https://www.h2o.ai/wp-content/themes/h2o2018/templates/dist/images/h2o_logo.svg"/>
```
There should definitely not be anything in this page sourced from the marketing website.  Make a copy in ./static

tomkraljevic commented on h2o-dist/index.html:
```diff
@@ -367,42 +370,56 @@
+            <img src="https://www.h2o.ai/wp-content/themes/h2o2018/templates/dist/images/h2o_logo.svg"/>
+          </a>
+          <p class="links">
+            <a href="http://h2o.ai/">H2O.ai</a> | <a href="https://github.com/h2oai/h2o-dev">Github</a>
```
This github URL is wrong.
It should be h2o-3, not h2o-dev.

tomkraljevic commented on h2o-dist/index.html:
```diff
@@ -367,42 +370,56 @@
-        <script type="text/javascript" src="ZeroClipboard.min.js"></script>
-        <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.0.0-alpha1/jquery.min.js"></script>
-        <script src="//app-ab12.marketo.com/js/forms2/js/forms2.min.js"></script>
+        <script type="text/javascript" src="./Download H2O 3.23.0.4496_files/ZeroClipboard.min.js"></script>
```
Putting build numbers like H2O 3.23.0.4496 is wrong.
Should be ./static

tomkraljevic commented on h2o-dist/index.html:
```diff
@@ -367,42 +370,56 @@
-        <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.0.0-alpha1/jquery.min.js"></script>
-        <script src="//app-ab12.marketo.com/js/forms2/js/forms2.min.js"></script>
+        <script type="text/javascript" src="./Download H2O 3.23.0.4496_files/ZeroClipboard.min.js"></script>
+        <script type="text/javascript" src="./Download H2O 3.23.0.4496_files/jquery.min.js"></script>
```
Should be ./static

tomkraljevic replied:
For this page though, that's not a good use model.

Think of this page as a standalone free-standing thing that still needs to work 3 years from now even if the dependencies vanish.  Anything you use should either come from inside this git repo or in the worst case, should live in an h2oai S3 bucket as a static artifact that will never be deleted.

This is not a typical marketing website that works one day and is gone the next.  This page needs to work forever.  Simpler is better, and deleting external dependencies is better.

spivotron replied:
I see. I've added them to ./static

tomkraljevic commented on h2o-dist/index.html:
```diff
@@ -367,42 +370,56 @@
-        <script src="//app-ab12.marketo.com/js/forms2/js/forms2.min.js"></script>
+        <script type="text/javascript" src="./Download H2O 3.23.0.4496_files/ZeroClipboard.min.js"></script>
+        <script type="text/javascript" src="./Download H2O 3.23.0.4496_files/jquery.min.js"></script>
+        <script src="./Download H2O 3.23.0.4496_files/forms2.min.js"></script>
```
Should be ./static

tomkraljevic commented on h2o-dist/index.html:
```diff
@@ -493,152 +512,161 @@ <h1><span>H<sub>2</sub>O</span></h1>
           <div id="quickstart-zip">
             <div id="downloads">
-              <a id="download-zip" class="download-button" href="h2o-SUBST_PROJECT_VERSION.zip">Download H<sub>2</sub>O</a>
+              <a id="download-zip" class="btn-yellow" href="http://h2o-release.s3.amazonaws.com/h2o/rel-xia/2/h2o-SUBST_PROJECT_VERSION.zip">Download H2O</a>
```
Don't have rel-xia references here.

tomkraljevic commented on h2o-dist/index.html:
```diff
@@ -493,152 +512,161 @@ <h1><span>H<sub>2</sub>O</span></h1>
+                    pip uninstall h2o<br>
+                    <br>
+                    <em># Next, use pip to install this version of the  H2O Python module.</em><br>
+                    pip install <script type="text/javascript">var url = location.href; var url2 = url.substring(0, url.lastIndexOf("/")); document.write(url2);</script>http://h2o-release.s3.amazonaws.com/h2o/rel-xia/2/Python/h2o-SUBST_PROJECT_VERSION-py2.py3-none-any.whl<br>
```
Don't have rel-xia references.

tomkraljevic commented on h2o-dist/index.html:
```diff
@@ -493,152 +512,161 @@ <h1><span>H<sub>2</sub>O</span></h1>
+                    }<br>
+                    <br>
+                    <em># Now we download, install and initialize the H2O package for R.</em><br>
+                    install.packages("h2o", type="source", repos="<script type="text/javascript">var url = location.href; var url2 = url.substring(0, url.lastIndexOf("/")); document.write(url2);</script>http://h2o-release.s3.amazonaws.com/h2o/rel-xia/2/R")<br>
```
Don't have rel-xia here.

This has shown up a lot of times now... Probably the right thing is to start over from the original file and only put in the stuff that's actually needed.

tomkraljevic commented on h2o-dist/index.html:
```diff
@@ -493,152 +512,161 @@ <h1><span>H<sub>2</sub>O</span></h1>
-                hadoop jar h2odriver.jar -nodes 1 -mapperXmx 6g<br/>
+                unzip h2o-SUBST_PROJECT_VERSION-*.zip<br>
+                cd h2o-SUBST_PROJECT_VERSION-*<br>
+                hadoop jar h2odriver.jar -nodes 1 -mapperXmx 6g -output hdfsOutputDirName<br>
```
Note the -output flag was removed.

spivotron replied:
Can you please clarify if the output flag is supposed to be removed?

tomkraljevic commented on h2o-dist/index.html:
```diff
@@ -493,152 +512,161 @@ <h1><span>H<sub>2</sub>O</span></h1>
+<br>
+repositories {<br>
+&nbsp;maven {<br>
+&nbsp;&nbsp;url "<script type="text/javascript">var url = location.href; var url2 = url.substring(0, url.lastIndexOf("/")); document.write(url2);</script>http://h2o-release.s3.amazonaws.com/h2o/rel-xia/2/maven/repo/"<br>
```
Remove rel-xia

spivotron commented:
This will be handled by @vineet-bhardwaj
